tests/angular-js/e2e/alternativeHelp.js

```diff
@@ -14,7 +14,7 @@
   var modalHeading = modal.element(by.css('h2'));
   var modalSubmit = modal.element(by.css('button[type="submit"]'));
 
-  describe('alternativeHelp', function () {
+  ddescribe('alternativeHelp', function () {
```

Comment:
Oops. That's 'run this test only' keyword that shouldn't be there. Well spotted!